src/couch_replicator/src/couch_replicator_scheduler.erl

```diff
@@ -138,11 +138,15 @@ job_summary(JobId, HealthThreshold) ->
             ErrorCount = consecutive_crashes(History, HealthThreshold),
             {State, Info} = case {Pid, ErrorCount} of
                 {undefined, 0}  ->
-                    {pending, null};
+                    case History of
```

Comment:
Since we're checking History in both instances of ErrorCount could we do it like this:

```
{undefined, _ErrorCount} ->
    case History of
         [{{crashed, Error}, _When} | _] ->
             {crashing, crash_reason_json(Error)};
         [_ | _] ->
             {pending, null}
    end,
```
?

Reply:
The difference is that when it is 0 we only show an error if the last event is an error, otherwise it is pending. So it is really a `pending` case mostly so I left it that way to make it clear to think about "pending, crashing and running".